modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -46,13 +50,23 @@ the associated capability.`),
 				return err
 			}
 
-			msg := types.NewMsgRegisterInterchainAccount(connectionID, owner, version)
+			orderString, err := cmd.Flags().GetString(flagOrdering)
+			if err != nil {
+				return err
+			}
+			order, err := parseOrder(orderString)
+			if err != nil {
+				return err
+			}
```

Comment:
[nit] maybe this can be a single function to return the `order` from the `flagOrdering`, seems like we don't need `orderString` for anything else.

Reply:
sgtm! Went ahead and consolidated the handling in the func.